src/main/java/seedu/recipe/logic/commands/ExportCommand.java

```diff
@@ -39,8 +39,12 @@ public ExportCommand(Stage stage, Logic logic) {
     public CommandResult execute(Model model) throws CommandException {
         try {
             exportManager.execute();
-        } catch (IOException e) {
-            throw new CommandException(ERROR_MESSAGE + e.getMessage());
+        } catch (IOException | CommandException e) {
+            if (e instanceof IOException) {
+                throw new CommandException(ERROR_MESSAGE + e.getMessage());
+            } else if (e instanceof CommandException) {
+                throw new CommandException(e.getMessage());
```

Comment:
not needed, just propagate the exception

Reply:
Right. Makes sense.